MPU/include/gpioHW.h

```diff
@@ -15,6 +15,8 @@ class PDU
 {
     private:
 
+        uint8_t motor_temp_to_rad_duty[20] = {0, 0, 0, 0, 0, 0, 25, 50, 75, 100, 128, 150, 200, 225, 255, 255, 255, 255, 255, 255};

```

Comment:
Is there no better way to do this lol? This feels very odd and unnecessary when we could implement a piecewise curve

Reply:
@mccauleyma